### PR TITLE
Add an easy way to handle other values of a choice type

### PR DESCRIPTION
**Pull request by carakas**
Once an other value has been added it will show up in the future options

### CODE REVIEW

**Comment by jonasdekeukelaere** [comment]
@carakas How do work with multiple languages?

**Comment by carakas** [comment]
@jonasdekeukelaere fair point, hadn't thought about that. Any suggestions?

**Comment by jonasdekeukelaere** [comment]
Maybe language in OtherChoiceOption and in OtherChoiceType only fetching those for current language?

File: app/migrations/Version20180222162856.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        $this->addSql('CREATE TABLE OtherChoiceOption (id CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', category VARCHAR(255) NOT NULL, `label` VARCHAR(255) NOT NULL, locale VARCHAR(2) NOT NULL, INDEX category_index (category), UNIQUE INDEX unique_name_index (category, `label`), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
+    }
+
+    public function down(Schema $schema)
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE User DROP FOREIGN KEY FK_2DA17977ED646567');
```
Comment (jonasdekeukelaere) [inline app/migrations/Version20180222162856.php]: Why?

Comment (carakas) [reply app/migrations/Version20180222162856.php]: probably something from the test code

Comment (carakas) [reply app/migrations/Version20180222162856.php]: fixed in
fa4a981

**Pull request merged**